packages/core/src/StateNode.ts

```diff
@@ -136,7 +134,10 @@ export class StateNode<
 
   protected __cache = {
     events: undefined as Array<TEvent['type']> | undefined,
-    relativeValue: new Map() as Map<StateNode<TContext>, StateValue>,
+    relativeValue: new Map() as Map<
```

Comment:
I've searched through the codebase and this seems to stay unused, some artifact of the past probably.